crates/bitwarden-core/src/platform/client_platform.rs

```diff
@@ -5,15 +5,54 @@ use super::{
 };
 use crate::{error::Result, Client};
 
+/// A struct containing platform utilities.
 pub struct ClientPlatform<'a> {
     pub(crate) client: &'a Client,
 }
 
 impl<'a> ClientPlatform<'a> {
+    /// Will generate a fingerprint based on the `input`. Given the same `input` This
+    /// method will result in the exact same output.
+    ///
+    /// # Examples
+    /// ```rust
+    /// use bitwarden_core::{Client, platform::FingerprintRequest};
+    ///
+    /// async fn test() {
+    ///     let client = Client::test_account().await;
+    ///     let fingerprint_response = client.platform()
+    ///         .fingerprint(&FingerprintRequest {
+    ///             fingerprint_material: "my_material".to_owned(),
+    ///             public_key: "...public key...".to_owned(),
+    ///         })
+    ///         .unwrap();
+    ///
+    ///     println!("{}", fingerprint_response.fingerprint);
```

Comment:
suggestion: give an example of the output
```suggestion
    ///     // Output: some-example
    ///     println!("{}", fingerprint_response.fingerprint);
```

Reply:
Was able to rewrite it to actually work and always output the same fingerprint so I changed it to an assert.